misc/scripts/dotnet_format.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import glob
+import os
+
+# Create dummy generated files.
+for path in [
```

Comment:
Sorry, last one, should we clean those files afterwards? I guess they're going to be overwritten on the next build, but IDK.

Reply:
Yes, they will be overwritten on the next build, so I don't know if it's worth it.

`SdkPackageVersions.props` is generated by `build_assemblies.py` so it will always be generated when building the C# projects anyway.

The `GeneratedIncludes.props` files are generated by `bindings_generator.cpp` so if someone tries to build the C# projects after running this script, without also re-generating the glue, the generated C# files won't be included in the compilation. Maybe we can skip writing these files if they already exist? Alternatively, https://github.com/godotengine/godot/pull/87130 would also _fix_ this.

Reply:
Alternatively, to avoid fiddling with the files for linting, we could condition the imports behind an environment var being unset. That way, the actual compilation step is untouched, but we can pass the var in environment before running `dotnet format`.
```csproj
<Import Condition=" '$(GODOT_SKIP_GENERATED)' == '' " Project="Generated\GeneratedIncludes.props" />
```

Reply:
That's a good idea, I'll make the import conditional. And we can decide on https://github.com/godotengine/godot/pull/87130 later.

Reply:
Then we shouldn't need to write those files here at all, right?

Reply:
I guess I could also apply the same condition to `SdkPackageVersions.props`.